{{cookiecutter.project_slug}}/Makefile

```diff
@@ -1,6 +1,8 @@
 SHELL=/bin/bash
 .DEFAULT_GOAL := help
-
+PROJECT_SLUG={{ cookiecutter.project_slug }}
```

Comment:
My OCD suggests that this one should be quoted if the other two are.

Reply:
That would break things like here: `./manage.py makemigrations --settings=${PROJECT_SLUG}.settings.migrations --check --dry-run`